bpf/lib/lb.h

```diff
@@ -152,13 +153,29 @@ static __always_inline int lb4_select_slave(__u16 count)
 }
 
 static __always_inline int extract_l4_port(struct __ctx_buff *ctx, __u8 nexthdr,
-					   int l4_off, __be16 *port)
+					   int l4_off, __be16 *port,
+					   __maybe_unused bool is_ipv4)
 {
 	int ret;
 
 	switch (nexthdr) {
 	case IPPROTO_TCP:
 	case IPPROTO_UDP:
+#if defined LB_4 && defined IPV4_FRAGMENTS && defined HAVE_LRU_MAP_TYPE
```

Comment:
Is this supposed to be like below?

```suggestion
#if defined LB_L4 && defined IPV4_FRAGMENTS && defined HAVE_LRU_MAP_TYPE
```

Reply:
:+1:
“Oops”